pkg/commands/secret/create_test.go

```diff
@@ -298,7 +338,7 @@ func testSecretCreateCommand(t *testing.T, when spec.G, it spec.S) {
 						defaultNamespacedServiceAccount,
 					},
 					Args: []string{secretName, "--git-url", gitRepo, "--git-user", gitUser, "-n", namespace},
-					ExpectedOutput: `Secret "my-git-basic-cred" created
+					ExpectedOutput: `Secret "my-git-basic-cred" created for registry https://github.com
```

Comment:
This output seems a bit awkward, can we get rid of the "registry" part? `Secret "my-git-basic-cred" created for https://github.com`